### PR TITLE
Test  Automation Framework Core and Utilities layers

### PR DESCRIPTION
**Pull request by Jengero**

### CODE REVIEW

File: TAF/TAF.Utilities/TAF.Utilities.csproj [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (MariaEroshevich) [inline TAF/TAF.Utilities/TAF.Utilities.csproj]: Use .Net7.

File: TAF/TestAutomationFramework/Browser/Browser.cs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using TAF.Helper;
+using TAF.Utilities.Logger;
+
+namespace TAF.Core.Browser
+{
+    public class Browser
+    {
```
Comment (MariaEroshevich) [inline TAF/TestAutomationFramework/Browser/Browser.cs]: Add ImplicitWait implementation and use it.

**Review by MariaEroshevich — CHANGES_REQUESTED** [review]

File: TAF/TAF.Utilities/TAF.Utilities.csproj (hunk 1 above)
Comment (Jengero) [reply TAF/TAF.Utilities/TAF.Utilities.csproj]: Resolved

File: TAF/TestAutomationFramework/Browser/Browser.cs (hunk 2 above)
Comment (Jengero) [reply TAF/TestAutomationFramework/Browser/Browser.cs]: Resolved

**Review by MariaEroshevich — APPROVED** [review]

**Pull request merged**